docs/step_by_step_guide.md

```diff
@@ -0,0 +1,646 @@
+# Useful Aliases
+Setting some aliases may be useful in running the model. You can edit your bash settings file to add aliases. Run `source` on that file to start using your new aliases. Examples:
+- Chrysalis – `source ~/.bashrc`
+- Compy – `source ~/.bash_profile`
+- Perlmutter – `source ~/.bash_profile.ext`
+
+Note that the specific file name may differ amongst machines. For example, it might be named `~/.bashrc`.ext.
+
+## Batch Jobs
+To check on all batch jobs: 
+
+`alias sqa='squeue -o "%8u %.7a %.4D %.9P %7i %.2t %.10r %.10M %.10l %.8Q %j" --sort=P,-t,-p'`
+
+To check on your batch jobs:
+
+`alias sq='sqa -u $USER'`
+
+The output of `sq` uses several abbreviations: ST = Status, R = running, PD = pending, CG = completing.
+
+## Directories
+You will be working in several directories. 
+
+<run_scripts_dir>: `${HOME}/E3SM/scripts`
+
+<code_source_dir>: `${HOME}/E3SM/code`
+
+<simulations_dir> differs amongst machines:
```

Comment:
These show up fine in the GitHub rendering, but in the actual rendering, anything in `<>` brackets simply shows up as an empty string...

Reply:
This problem recurs throughout the file.